docs/maui/markup/theming.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Theming - .NET MAUI Community Toolkit
+author: bijington
+description: C# Markup is a set of fluent helper methods and classes designed to simplify the process of building declarative .NET Multi-platform App UI (.NET MAUI) user interfaces in code.
```

Comment:
Is this the description we use for every page? Should it be more specific to theming in this case?